FEEDBACK.md

```diff
@@ -0,0 +1,334 @@
+
+You can use the popover mode to implement your own button to collect feedback manually.
+
+```javascript
+position: {
+  type: "POPOVER",
+  anchor: DOMElement
+}
+```
+
+Popover feedback button example:
+
+```html
+<button id="feedbackButton">Tell us what you think</button>
+<script>
+  const button = document.getElementById("feedbackButton");
+  button.addEventListener("click", (e) => {
+    bucket.requestFeedback({
+      featureId: "bucket-feature-id",
+      userId: "your-user-id",
+      title: "How do you like the popover?",
+      position: {
+        type: "POPOVER",
+        anchor: e.currentTarget,
+      },
+    });
+  });
+</script>
+```
+
+## Internationalization (i18n)
+
+The Feedback UI currently only comes in English. You can supply your own translation keys that will be used insude the UI by passing an object with your own translation keys.
+
+See [default english localization keys](./src/feedback/config/defaultTranslations.tsx) for a reference of what translation keys can be supplied.
+
+### Static language configuration
+
+If you know the language at page load, you can configure your translation keys while initializing the Bucket SDK:
+
+```javascript
+bucket.init("my-tracking-key", {
+  feedback: {
+    ui: {
+      translations: {
+        // Your translation keys
+      },
+    },
+  },
+});
+```
+
+### Runtime language configuration
+
+If you only know the users language after the page has loaded, you can provide translation keys in the two cases where you open the feedback UI:
+
+### Manual feedback collection
+
+```javascript
+bucket.requestFeedback({
+  ... // Other options
+  translations: {
+    // your translation keys
+  }
+})
+```
+
+### Automated feedback collection
+
+When you are collecting feedback through the Bucket automation, you can intercept the default prompt handling and override the defaults.
+
+If you set the prompt question in the Bucket app to be one of your own translation keys, you can even get a translated version of the question you want to ask your customer in the feedback UI.
+
+```javascript
+bucket.init({
+  feedback: {
+    promptHandler: (message, handlers) => {
+      const translatedQuestion =
+        i18nLookup[message.question] ?? message.question;
+      handlers.openFeedbackForm({
+        title: translatedQuestion,
+        translations: {
+          // your static translation keys
+        },
+      });
+    },
+  },
+});
+```
+
+## Custom styling
+
+You can adapt parts of the looks of the Bucket feedback UI by setting CSS custom properties on your CSS `:root`-scope.
+
+Examples of custom styling can be found in our [development example stylesheet](./dev/index.css).
+
+## Using your own UI to collect feedback
+
+You might have very strict design guidelines in your app, and maybe the Bucket feedback UI doesn"t quite work for you.
+
+In this case, you can actually implement your own feedback collection mechanism, which follows your own design guidelines.
```

Comment:
I suggest:

In this case, you can implement your own feedback collection mechanism, which follows your own design guidelines.

Reply:
```suggestion
In this case, you can implement your own feedback collection mechanism, which follows your own design guidelines.
```